### DISCUSSION TITLE
Washer Dryer Notifications

### DISCUSSION THREAD

**Discussion by JSVarga** [nVuln/homebridge-lg-thinq#88] [General]
My use case is our old machine stayed on until unloaded, however our LG washer dryer auto turns off very soon and sometimes we have left loads in there after washing. The notification, using the occupancy sensor is useful however it is only a one time notification and the occupancy sensor stays on although it sometimes seems to go off.
Does the occupancy sensor have a default time out? Is there a way of forcing the occupancy sensor to off or is that all internally managed?
I have considered using dummy switches to trigger notifications on a periodic basis but they don't currently support iOS notifications for which I have submitted an issue/enhancement. Does anyone have any other ideas?
Basically trying to get periodic iOS notifications until we actually unload the machine.

**Comment by nVuln** [comment]
please try [homebridge-http-switch](https://www.npmjs.com/package/homebridge-http-switch) and see [Homebridge UI API](https://github.com/oznu/homebridge-config-ui-x/wiki/API-Reference)
I think you can force sensor status via homebridge rest api
